metashare/api/jobs.py

```diff
@@ -470,3 +471,39 @@ def populate_github_users(repository):
 
 
 populate_github_users_job = job(populate_github_users)
+
+
+def submit_review(*, user, task, data):
+    try:
+        task.refresh_from_db()
+        repository = get_repo_info(user, repo_id=task.project.repository.repo_id)
+        pr = repository.pull_request(task.pr_number)
+        # We always COMMENT so as not to change the PR's status:
+        pr.create_review(data["notes"], event="COMMENT")
```

Comment:
@wlonk Out of curiosity, what will this do if `pr_number` is `None`?

Reply:
Sort of addressed in https://github.com/oddbird/MetaShare/pull/164/commits/a758fc098e144db867c390ad4926f7bdecd667ab